### PR TITLE
Feature/create route that handle request data and add new airtable-posts row

### PR DESCRIPTION
**Pull request by mhmdtshref**
Related #90

### CODE REVIEW

File: config.env.example [hunk 1]
```diff
@@ -1 +1,2 @@
-CODECOV_TOKEN=""
+CODECOV_TOKEN="[ADD YOUR CODECOV TOKEN HERE]"
```
Comment (teenie-quaggard) [inline config.env.example]: I love this config.env example!
I have never done that before, but it makes so much sense!

File: package.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
   },
   "homepage": "https://github.com/founders-programme-2/howl#readme",
   "dependencies": {
+    "airtable": "^0.6.0",
```
Comment (teenie-quaggard) [inline package.json]: Can you please add airtable to the 'stack' part of the readme, with a blurb about what it does and a link to it (you can copy the format that I've made for cloudinary and the otheres)? Thank you! It's always nice to document as we go, so it's not a gigantic task at the end :)

File: src/airtables/index.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+const Story = require('./story');
```
Comment (teenie-quaggard) [inline src/airtables/index.js]: please update to es6

File: src/airtables/index.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
...
+
+module.exports = { Story };
```
Comment (teenie-quaggard) [inline src/airtables/index.js]: please update to es6

File: src/airtables/index.js (hunk 3 above)
Comment (teenie-quaggard) [reply src/airtables/index.js]: ```import Story from './story'```

File: src/airtables/index.js (hunk 4 above)
Comment (teenie-quaggard) [reply src/airtables/index.js]: ```export default Story```

File: src/airtables/story.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+const Airtable = require('airtable');
+require('env2')('config.env');
+
+const { AIRTABLE_API_KEY } = process.env;
+
+if (!AIRTABLE_API_KEY) {
+  throw new Error('Missing Airtable API KEY env var');
+}
+
+const base = new Airtable({ apiKey: AIRTABLE_API_KEY }).base('apppdW6Ebu2yFkm80');
```
Comment (teenie-quaggard) [inline src/airtables/story.js]: just a question - what is the ```.base``` ?

File: src/airtables/story.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+const Airtable = require('airtable');
```
Comment (teenie-quaggard) [inline src/airtables/story.js]: please convert to es6

File: src/airtables/story.js [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+
+const { AIRTABLE_API_KEY } = process.env;
+
+if (!AIRTABLE_API_KEY) {
+  throw new Error('Missing Airtable API KEY env var');
+}
+
+const base = new Airtable({ apiKey: AIRTABLE_API_KEY }).base('apppdW6Ebu2yFkm80');
+
+module.exports = base('posts');
```
Comment (teenie-quaggard) [inline src/airtables/story.js]: please convert to es6

File: src/controllers/story.js [hunk 8]
```diff
@@ -0,0 +1,22 @@
+const { Story } = require('../airtables');
```
Comment (teenie-quaggard) [inline src/controllers/story.js]: convert to es6

File: src/controllers/story.js [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+  } = req.body;
+
+  Story.create({
+    owner,
+    title,
+    body,
+    date,
+  }, (err) => {
+    if (err) {
+      res.json({ success: false, err: 'Saving data error' });
```
Comment (teenie-quaggard) [inline src/controllers/story.js]: Small thing, can we change the error to be a little bit more descriptive? The syntax makes it a bit hard to understand and I might be confused if I received it!

Maybe, 'There's been an error in saving the data to Airtable'

File: src/controllers/story.js [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+    } else {
+      res.json({ success: true });
+    }
+  });
+};
+
+module.exports = { create };
```
Comment (teenie-quaggard) [inline src/controllers/story.js]: please use es6 import/export

File: src/router.js [hunk 11]
```diff
@@ -7,5 +7,6 @@ const { uploadImg, multerUploads, cloudinaryConfig } = require('./middlewares/up
 
 router.post('/uploadImg', [cloudinaryConfig, multerUploads, uploadImg]);
 
+router.post('/posts/create', require('./controllers/story').create);
 
 module.exports = router;
```
Comment (teenie-quaggard) [inline src/router.js]: es6 please!

File: src/airtables/index.js (hunk 4 above)
Comment (mhmdtshref) [reply src/airtables/index.js]: Done!

File: src/airtables/story.js (hunk 5 above)
Comment (mhmdtshref) [reply src/airtables/story.js]: I found it on the airtable api documentation, it's looks like that every airtable account is having code like this. but I don't think that it's a secret code.

File: src/controllers/story.js (hunk 9 above)
Comment (mhmdtshref) [reply src/controllers/story.js]: Done!

File: src/controllers/story.js (hunk 10 above)
Comment (mhmdtshref) [reply src/controllers/story.js]: Done!

File: src/router.js (hunk 11 above)
Comment (mhmdtshref) [reply src/router.js]: Done!

File: src/controllers/story.js (hunk 8 above)
Comment (mhmdtshref) [reply src/controllers/story.js]: Done!

File: src/airtables/story.js (hunk 7 above)
Comment (mhmdtshref) [reply src/airtables/story.js]: Done!

File: src/airtables/story.js (hunk 6 above)
Comment (mhmdtshref) [reply src/airtables/story.js]: Done!

File: package.json (hunk 2 above)
Comment (mhmdtshref) [reply package.json]: Done!

File: src/airtables/index.js (hunk 3 above)
Comment (mhmdtshref) [reply src/airtables/index.js]: Done!

**Pull request merged**